functions/index.js

```diff
@@ -142,7 +191,7 @@ function toDataByLocation(data) {
     const entry_obj = {};
     headers.forEach( (value, index) => {
       if (entry[index] !== undefined) {
-        entry_obj[value] = entry[index].trim()
+        entry_obj[value] = (typeof entry[index]) === 'string' ? entry[index].trim() : entry[index];
```

Comment:
when is this even not a string??

Reply:
For a column containing a lat/lng value - it has numeric type in that case, so calling trim() on it fails.